controllers/defaults.go

```diff
@@ -70,25 +70,46 @@ var (
 
 	// CSI Controller resource requests/limits
 	// TODO: Reduce these values and reach optimistic values without effecting performance
-	TopolvmControllerMemRequest = "250Mi"
-	TopolvmControllerMemLimit   = "250Mi"
-	TopolvmControllerCPURequest = "250m"
-	TopolvmControllerCPULimit   = "250m"
+	TopolvmControllerMemRequest = "100Mi"
+	TopolvmControllerMemLimit   = "150Mi"
+	TopolvmControllerCPURequest = "100m"
+	TopolvmControllerCPULimit   = "100m"
 
-	TopolvmCsiProvisionerMemRequest = "100Mi"
+	TopolvmCsiProvisionerMemRequest = "50Mi"
 	TopolvmCsiProvisionerMemLimit   = "100Mi"
-	TopolvmCsiProvisionerCPURequest = "100m"
+	TopolvmCsiProvisionerCPURequest = "50m"
 	TopolvmCsiProvisionerCPULimit   = "100m"
 
-	TopolvmCsiResizerMemRequest = "100Mi"
+	TopolvmCsiResizerMemRequest = "50Mi"
 	TopolvmCsiResizerMemLimit   = "100Mi"
-	TopolvmCsiResizerCPURequest = "100m"
-	TopolvmCsiResizerCPULimit   = "100m"
+	TopolvmCsiResizerCPURequest = "20m"
+	TopolvmCsiResizerCPULimit   = "50m"
 
-	TopolvmCsiSnapshotterMemRequest = "100Mi"
+	TopolvmCsiSnapshotterMemRequest = "50Mi"
 	TopolvmCsiSnapshotterMemLimit   = "100Mi"
-	TopolvmCsiSnapshotterCPURequest = "100m"
-	TopolvmCsiSnapshotterCPULimit   = "100m"
+	TopolvmCsiSnapshotterCPURequest = "20m"
+	TopolvmCsiSnapshotterCPULimit   = "50m"
+
+	VgManagerMemRequest = "50Mi"
+	VgManagerMemLimit   = "100Mi"
+	VgManagerCPURequest = "30m"
+	VgManagerCPULimit   = "50m"
+
+	// topoLVM Node resource requests/limits
+	TopolvmNodeMemRequest = "100Mi"
+	TopolvmNodeMemLimit   = "150Mi"
+	TopolvmNodeCPURequest = "100m"
+	TopolvmNodeCPULimit   = "150m"
+
```

Comment:
Please add separate values for the lvmd. The topolvm-node container uses a lot less. Lvmd can use a request val of 150m and limit of 200m.

Reply:
add separate values for lvmd and topolvm-node. 
But topolvm-node uses much more when compared to lvmd.  For example:
``` Report for topolvm-node daemonset's Pod topolvm-node-5jb5m between 2022-06-21 12:35:20 +0530 IST and 2022-06-21 12:52:12 +0530 IST
	 CPU (min|max|avg) seconds:     0.0019 |     0.2090 |     0.1030
	 MEM (min|max|avg) Mib    :   100.8945 |   137.1836 |   120.3202```

``` 
Report for topolvm-node daemonset's Container lvmd between 2022-06-21 12:35:20 +0530 IST and 2022-06-21 12:52:12 +0530 IST
	 CPU (min|max|avg) seconds:     0.0000 |     0.1910 |     0.0933
	 MEM (min|max|avg) Mib    :    31.9258 |    62.1250 |    47.2663
```

Reply:
The other results (40 pods, 45 pods) show consistently lower values for the topolvm-node vs the lvmd containers.

Reply:
my bad. I was looking at the entire pod. :/ 
What do you think about the current resource allocation for the lvmd and topolvm-node container ?

Reply:
The lvmd container looks fine. The topolvm-node can be reduces to 50 for request and 100 for limit.

Reply:
The lvm-operator manager cpu numbers can be reduced as well - will it work with say 50 and 100?

Reply:
Please raise the mem request for lvm-operator  - lets make it at least 50.